.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+## Before I have submitted the issue 
+[ ] I have read an [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html)  
+[ ] I know that for an iOS I need to install pods because I've read the [installation guide](https://tipsi.github.io/tipsi-stripe/docs/installation.html#cocoapods-ios)  
+[ ] I have read a [linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html) and check that everything is OK like in [manual linking guide](https://tipsi.github.io/tipsi-stripe/docs/linking.html#manual)  
+[ ] I know that before using `tipsi-stripe` I need to set options for my app like described in [usage guide](https://tipsi.github.io/tipsi-stripe/docs/usage.html)  
+
+## The problem
+Briefly describe the issue you are experiencing (or the feature you want to see added to `tipsi-stripe`). 
+Tell us what you were trying to do and what happened instead. 
+
+## Environment
+* `tipsi-stripe` version:
+* Last `tipsi-stripe` version that did not exhibit the issue (if applicable):
```

Comment:
Last `tipsi-stripe` version that did not exhibit the issue (if applicable)->Last `tipsi-stripe` version where the issue was not reproduced (if applicable)